vip-helpers/class-vip-backup-user-role-cli.php

```diff
@@ -28,6 +28,12 @@ class VIP_Backup_User_Role_CLI extends \WPCOM_VIP_CLI_Command {
 	public function list_( $args, $assoc_args ) {
 
 		$backup_roles = get_option( 'vip_backup_user_roles', [] );
+		if ( ! is_array( $backup_roles ) ) {
+			\WP_CLI::warning( 'The option vip_backup_user_roles is not an array' );
+			$debug_export = var_export( $backup_roles, true );
+			\WP_CLI::error_multi_line( $debug_export );
+			\WP_CLI::error( 'exit as nothing to do' );
```

Comment:
This line and similar below seem a bit redundant, I think silently bailing is enough?

Reply:
I can replace with an exit call